flightdata-kafka-consumer/src/main/java/com/jackdaw/consumer/JSONSerializer.java

```diff
@@ -0,0 +1,160 @@
+package com.jackdaw.consumer;
+
+import com.jackdaw.avro.flights.Flight;
+import com.jackdaw.avro.flights.TimeType;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class JSONSerializer {
+    public void write(String filename, Flight flight) {
+        JSONObject featureCollection = new JSONObject();
+        JSONArray featureList = new JSONArray();
+        JSONObject feature = new JSONObject();
+        JSONObject geometry = getCorrectGeometry(flight);
+        JSONObject properties = getProperties(flight);
+
+        featureCollection.put(TYPE, FEATURE_COLLECTION);
+        feature.put(TYPE, FEATURE);
+        feature.put(GEOMETRY, geometry);
+        feature.put(PROPERTIES, properties);
+        featureList.put(feature);
+        featureCollection.put(FEATURES, featureList);
+
+        try {
+            FileWriter fw = new FileWriter("/volume/" + filename, true);
+            fw.write(featureCollection.toString(4));
+            fw.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private JSONObject getCorrectGeometry(Flight flight) {
+        if (flight.getTimeType() == TimeType.arrivalEstimate) {
+            return getLineString(flight);
+        }
+        return getPoint(flight);
+    }
+
+    private JSONObject getPoint(Flight flight) {
+        JSONObject point = new JSONObject();
+        JSONArray coordinates = new JSONArray();
+
+        coordinates.put(flight.getOriginAltitude());
+        coordinates.put(flight.getOriginLongitude());
+
```

Comment:
Depending on timeType you should return OriginAltitude/Longitude (for real/estimateDeparture) or DestinationAltitude/Longitude (for realArrival)